### PR TITLE
Fix issues of ubuntu branch weekly job on RELEASE-3.1.x

### PR DESCRIPTION
**Pull request by ivy275**
Fix defect 384858:
https://jazzop27.rtp.raleigh.ibm.com:9443/ccm/web/projects/BlueBox%20Cloud#action=com.ibm.team.workitem.viewWorkItem&id=384858

Includes:
1. Fixed issue in task "neutron-control : sync neutron database for lbaas" by changing cmd to "neutron-db-manage --subproject neutron-lbaas --config-file /etc/neutron/neutron.conf --config-file /etc/neutron/plugins/ml2/ml2_plugin.ini upgrade head".

2. Synced rally related tasks with master branch.

3. Synced testenv_instance_prefix in test/common with master branch.

Job ran locally passed.

### CODE REVIEW

File: roles/neutron-control/tasks/main.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@
   tags: db-migrate
 
 - name: sync neutron database for lbaas
-  command: neutron-db-manage --service lbaas --config-file /etc/neutron/neutron.conf \
+  command: neutron-db-manage --subproject neutron-lbaas --config-file /etc/neutron/neutron.conf \
```
Comment (nirajdp76) [inline roles/neutron-control/tasks/main.yml]: why we need this change? We just completed QA and UAT of 3.1.1.3 and did not see any issues around neutron db sync

**Comment by nirajdp76** [comment]
Are these changes already on master? We cannot make changes to just release branch. Typical flow is PR to master and cherry pick to branch. If there is strong reason to have fix made only to release branch than that must be mentioned.

**Comment by nirajdp76** [comment]
retest

File: roles/neutron-control/tasks/main.yml (hunk 1 above)
Comment (ivy275) [inline roles/neutron-control/tasks/main.yml]: Tested RELEASE-3.1.x with package (2016.1-bbc248) in local and met the issue, but don't meet it in master branch:

2017-04-12 09:10:41,854 p=25485 u=ubuntu |  TASK [neutron-control : sync neutron database for lbaas] ***********************
2017-04-12 09:10:41,854 p=25485 u=ubuntu |  Wednesday 12 April 2017  09:10:41 +0000 (0:00:06.934)       0:41:19.461 *******
2017-04-12 09:10:44,331 p=25485 u=ubuntu |  fatal: [controller-0]: FAILED! => {"changed": true, "cmd": ["neutron-db-manage", "--service", "lbaas", "--config-file", "/etc/neutron/neutron.conf", "--config-file", "/etc/neutron/plugins/ml2/ml2_plugin.ini", "upgrade", "head"], "delta": "0:00:01.673791", "end": "2017-04-12 09:10:44.056173", "failed": true, "rc": 2, "start": "2017-04-12 09:10:42.382382", "stderr": "usage: neutron-db-manage [-h] [--config-dir DIR] [--config-file PATH]\n                         [--nosplit_branches] [--split_branches]\n                         [--subproject SUBPROJECT]\n                         [--database-connection DATABASE_CONNECTION]\n                         [--database-engine DATABASE_ENGINE]\n                         {current,history,branches,heads,check_migration,upgrade,downgrade,stamp,revision,has_offline_migrations}\n                         ...\nneutron-db-manage: error: argument command: invalid choice: 'lbaas' (choose from 'current', 'history', 'branches', 'heads', 'check_migration', 'upgrade', 'downgrade', 'stamp', 'revision', 'has_offline_migrations')", "stdout": "", "stdout_lines": [], "warnings": []}

**Comment by ivy275** [comment]
@nirajdp76, There are three parts will be updated to RELEASE-3.1.x branch in this PR:
1. For test/common and test/setup, all changes are copied from master branch to keep testenv_instance_prefix correctly, etc.
2. For rally stuff, also copied from master branch.
3. For change in roles/neutron-control/tasks/main.yml, I compared RELEASE-3.1x and master branch, and found that task "sync neutron database for lbaas" only existing on RELASE-3.1.x, that's why we only need modify in RELEASE-3.1.x but not in master branch.

I debugged locally based on these changes for RELEASE-3.1.x branch with package (2016.1-bbc248) installed, passed.

File: roles/neutron-control/tasks/main.yml (hunk 1 above)
Comment (ivy275) [inline roles/neutron-control/tasks/main.yml]: Well, I rolled back the file to use "--service lbaas" and tried to run again, and found this issue is gone.. So weird...
Not sure what happened, but anyway, I will continue to use "--service lbaas".

**Comment by ivy275** [comment]
Remove point #3 "For change in roles/neutron-control/tasks/main.yml", since cmd with "--service lbaas" works again.

**Comment by lihkin213** [comment]
Also please remove internal bug tracking references from the description.

**Pull request merged**